src/provider/element-templates/ElementTemplates.js

```diff
@@ -14,14 +14,21 @@ import {
   getTemplateVersion
 } from './Helper';
 
+import { removeTemplate as removeTemplateHelper } from './util/templateUtil';
+
 import { isAny } from 'bpmn-js/lib/features/modeling/util/ModelingUtil';
 
 /**
  * Registry for element templates.
  */
 export default class ElementTemplates {
-  constructor(commandStack) {
+  constructor(commandStack, replace, selection, eventBus, modeling, injector) {
     this._commandStack = commandStack;
+    this._replace = replace;
+    this._selection = selection;
```

Comment:
We don't seem to use `selection` and `replace`.

Reply:
This is leftover because the logic that uses this was moved into `removeTemplateHelper`